### PR TITLE
display alphabetical group when clicked (pagination)

### PR DESCRIPTION
**Pull request by berlintam**
retry for https://github.com/rubycorns/rorganize.it/pull/512

slight problem is, if we have an alphabetical list (from A-Z) and go to /people, we display the people whose names start with Z first, not the ppl whose names start with A. 

@sareg0 is gonna take another look at this! suggestions welcome on how to get the page to display at A group first :)

(ex: in the screenshot you can see that if you go to /people the last alphabetical group of people is shown first)

screenshot from 2017-09-26 21-57-23

### CODE REVIEW

File: app/views/people/index.html.haml [hunk 1]
```diff
@@ -11,3 +11,10 @@
 
   %aside.flex-index-sidebar
     = render partial: 'shared/filter', locals: { path: people_path }
+
+
+%script
+  :plain
```
Comment (til) [inline app/views/people/index.html.haml]: There is also a `:javascript` thing - filter I think it is called - in haml, which outputs a script tag automatically.

File: app/views/people/index.html.haml [hunk 2]
```diff
@@ -11,3 +11,10 @@
...
+  if (window.location.hash === '') {
+    window.location.hash = "#{@alphabetical_list.first}";
+  }
```
Comment (til) [inline app/views/people/index.html.haml]: Also, not only GitHub prefers a file to end in a newline

**Comment by berlintam** [comment]
Thanks for taking this over @sareg0!! :green_heart:

**Comment by sareg0** [comment]
thanks @til! and @berlintam you're welcome! Just filling in the lil' bug you were having to close this puppy up! 🐩

**Comment by lislis** [comment]
puppies

**Comment by til** [comment]
Time to mööörrrge this!!

**Comment by berlintam** [comment]
excite!

**Pull request merged**